### PR TITLE
Add overview report (#82)

### PR DESCRIPTION
**Pull request by dsotirho-ucsc**
For #82

### CODE REVIEW

**Comment by danielsotirhos** [comment]
For #82

File: overview_report.py [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+    matrices_count: int = 0  # number of matrices in projects/{uuid}/matrices
+    zipped_matrix: Path = None  # projects/{uuid}/bundle/matrix.mtx.zip
+    cell_count: int = 0  # value of cell count in cell_counts.json
+    metadata_json_count: int = 0  # number of metadata JSON files in projects/{uuid}/bundle
+
+    @classmethod
+    def tsv_header(cls) -> Sequence[str]:
+        return [
+            'UUID',
+            'Accession ID',
```
Comment (hannes-ucsc) [inline overview_report.py]:
```suggestion
            'accession',
```

Accessions *are* identifiers.

File: overview_report.py [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+            'Project path',
```
Comment (hannes-ucsc) [inline overview_report.py]:
```suggestion
            'project_path',
```

File: overview_report.py [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+            'Accession symlink',
```
Comment (hannes-ucsc) [inline overview_report.py]:
```suggestion
            'symlink',
```

File: overview_report.py [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+            'Spreadsheet',
+            'Geo files count',
```
Comment (hannes-ucsc) [inline overview_report.py]:
```suggestion
            'geo_files',
```

File: overview_report.py [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+    geo_files_count: int = 0  # number of downloaded geo files in projects/{uuid}/geo
+    matrices_count: int = 0  # number of matrices in projects/{uuid}/matrices
+    zipped_matrix: Path = None  # projects/{uuid}/bundle/matrix.mtx.zip
+    cell_count: int = 0  # value of cell count in cell_counts.json
+    metadata_json_count: int = 0  # number of metadata JSON files in projects/{uuid}/bundle
+
+    @classmethod
+    def tsv_header(cls) -> Sequence[str]:
+        return [
+            'UUID',
```
Comment (hannes-ucsc) [inline overview_report.py]:
```suggestion
            'uuid',
```

File: overview_report.py [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+            'Accession ID',
+            'Project path',
+            'Accession symlink',
+            'Spreadsheet',
```
Comment (hannes-ucsc) [inline overview_report.py]:
```suggestion
            'spreadsheet',
```

File: overview_report.py [hunk 7]
```diff
@@ -0,0 +1,249 @@
...
+            'Geo files count',
+            'Matrices count',
```
Comment (hannes-ucsc) [inline overview_report.py]:
```suggestion
            'num_matrices',
```

File: overview_report.py [hunk 8]
```diff
@@ -0,0 +1,249 @@
...
+            'Zipped matrix',
```
Comment (hannes-ucsc) [inline overview_report.py]:
```suggestion
            'zipped_matrix',
```

File: overview_report.py [hunk 9]
```diff
@@ -0,0 +1,249 @@
...
+            'Cell count',
```
Comment (hannes-ucsc) [inline overview_report.py]:
```suggestion
            'cell_count',
```

File: overview_report.py [hunk 10]
```diff
@@ -0,0 +1,249 @@
...
+            'Metadata JSON count',
```
Comment (hannes-ucsc) [inline overview_report.py]:
```suggestion
            'num_metadata_files',
```

File: Makefile [hunk 11]
```diff
@@ -31,4 +31,7 @@ bundles:
 assets:
 	python3 upload_assets.py
 
+report:
+	python3 overview_report.py
```
Comment (hannes-ucsc) [inline Makefile]:
```suggestion
	python3 overview_report.py | column -nt | less -S
```

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]
It would be more flexible if this wrote to stdout. See also my suggestion on the make file.

**Review by hannes-ucsc — APPROVED** [review]

**Pull request merged**